Sources/ArgumentParser/Utilities/Tree.swift

```diff
@@ -90,7 +90,12 @@ extension Tree where Element == ParsableCommand.Type {
   
   convenience init(root command: ParsableCommand.Type) throws {
     self.init(command)
+    var subcommandNames = Set<String>()
     for subcommand in command.configuration.subcommands {
+      if subcommandNames.contains(subcommand._commandName) {
+        throw InitializationError.matchingCommandNames(command, subcommand._commandName)
+      }
+      subcommandNames.insert(subcommand._commandName)
```

Comment:
Tiny note, insert returns a Bool indicating if the new value was inserted, you can use this instead of checking contains

Reply:
A beautiful API 🙏, will change to this

Reply:
Done!